### ISSUE TITLE
Error when remove user from group

### ISSUE THREAD

**Issue by Imploxion** [brainly/terraform-provider-redshift#83]
I am trying to remove some users from a group but when run the apply I am getting an error from the provider that cannot find the user info.
I tried with different groups and users getting the same result. Not sure if there is a problem with the query in [checkIfUserExists](https://github.com/brainly/terraform-provider-redshift/blob/0c991525890daeb965402332da416214d1ce5121/redshift/resource_redshift_group.go#L222)

```
module.aws-redshift.redshift_group.this["group_name"]: Modifying... [id=44633053]
2022-09-16T13:34:16.670+0200 [ERROR] vertex "module.aws-redshift.redshift_group.this[\"group_name\"]" error: error reading info about user: %!s(<nil>)
╷
│ Error: error reading info about user: %!s(<nil>)
│
│   with module.aws-redshift-data-redshift.redshift_group.this["group_name"],
│   on modules/aws-redshift/main.tf line 85, in resource "redshift_group" "this":
│   85: resource "redshift_group" "this" {
│
╵
´´´

**Comment by winglot** [comment]
Which version of the provider are you using?

I believe this has been fixed in the latest v1.0.2 release (https://github.com/brainly/terraform-provider-redshift/releases/tag/v1.0.2)

**Comment by Imploxion** [comment]
I was using 1.0.1, I have updated and it worked.
Thank you
